desc/backend.py

```diff
@@ -1,6 +1,7 @@
 """Backend functions for DESC, with options for JAX or regular numpy."""
 
 import functools
+import multiprocessing as mp
```

Comment:
is this needed? I thought you got rid of all the multiprocessing stuff?

If it's just to avoid printing extra stuff I'd rather not import it, sometimes just importing multiprocessing and jax can lead to random (mostly benign, but potentially confusing) warnings.

Reply:
There is no multiprocessing stuff in this PR, so I could move these changes to #1046. It is only to avoid printing extra stuff, but I argue that it is needed. Because otherwise with the TERPSICHORE objective (and any others that use multiprocessing) the optimizer output becomes illegible. It will print this backend stuff of "DESC version ... Using device ..." hundreds of times between each optimizer iteration.

Reply:
make env info a standalone function, and do not auto-print it upon import